### PR TITLE
Add link to skip to main content 

### PR DESCRIPTION
**Pull request by laurakwhit**
## Description & motivation 💭 
Adds a "Skip to main content" link that is accessible via keyboard.

### Screenshots (if applicable) 📸 
Screenshot 2023-08-31 at 11 55 15 AM

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
- [ ] Verify the "Skip to main content" link places focus in main content
- [ ] Verify the "Skip to main content" link is only visible when focused

### Issue(s) closed 
`DT-1228`

### CODE REVIEW

**Comment by Alex-Tideman** [comment]
How do you view this? What's the keyboard shortcut?

**Comment by Alex-Tideman** [comment]
Nvm I see it when I tab.

**Comment by laurakwhit** [comment]
@Alex-Tideman it should be the first thing you tab to when you load the page.

**Comment by Alex-Tideman** [comment]
Looks like this doesn't work in Safari or FF. I don't see it when I tab.

File: src/lib/components/skip-nav.svelte [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+  import Link from '$lib/holocene/link.svelte';
+  import { translate } from '$lib/i18n/translate';
+</script>
+
+<Link
+  id="skip-nav"
+  href="#content"
+  class="absolute left-1/2 translate-y-[-100%] focus:translate-y-0 z-[100] p-1"
```
Comment (stevekinney) [inline src/lib/components/skip-nav.svelte:9]: Why not `translate-y-full`?

Are we okay with the bespoke z-index?

File: src/routes/(app)/+layout.svelte [hunk 2]
```diff
@@ -8,6 +8,7 @@
   import Banners from '$lib/components/banner/banners.svelte';
   import DataEncoderSettings from '$lib/components/data-encoder-settings.svelte';
   import SideNavigation from '$lib/components/side-nav.svelte';
+  import SkipNav from '$lib/components/skip-nav.svelte';
```
Comment (stevekinney) [inline src/routes/(app)/+layout.svelte:11]: Can I be pendantic and ask that we call this `SkipNavigation`?

**Comment by laurakwhit** [comment]
@Alex-Tideman have you turned on `Full Keyboard Access` in `System Settings` > `Accessibility` > `Keyboard`?

File: src/lib/components/skip-nav.svelte (hunk 1 above)
Comment (laurakwhit) [reply src/lib/components/skip-nav.svelte:9]: > Why not translate-y-full?

Needs to be a negative (we want it off the page).

> Are we okay with the bespoke z-index?
We can update this to `z-50`, but then we'll need to update `NavContainer` to something else (and potential a few other things). I was 50/50 and so went with bespoke.

File: src/routes/(app)/+layout.svelte (hunk 2 above)
Comment (laurakwhit) [reply src/routes/(app)/+layout.svelte]: [Done](https://github.com/temporalio/ui/pull/1611/commits/741522d878311c87d8d05dbdf699407f1f6e15db) ✅

File: src/lib/components/skip-nav.svelte (hunk 1 above)
Comment (stevekinney) [reply src/lib/components/skip-nav.svelte:9]: Oh, I can't read. Why not `-translate-y-full`?

Comment (stevekinney) [reply src/lib/components/skip-nav.svelte:9]: Yea, I think maybe we should just update to standardized z-indices.

Comment (laurakwhit) [reply src/lib/components/skip-nav.svelte:9]: > Why not -translate-y-full?

TIL

> update to standardized z-indices.

Opinions on what these might be? Or perhaps I should just add 100 to the config for now...

**Review by Alex-Tideman — APPROVED** [review]

**Pull request merged**